packages/velo-external-db-core/lib/service/schema_matchers.js

```diff
@@ -9,10 +9,20 @@ const appendAllowedOperationsToDbs = (dbs, allowedSchemaOperations) => {
     }))
 }
 
-const toHaveSchemas = ( dbsWithAllowedOperations ) => ({ schemas: dbsWithAllowedOperations.map(asWixSchema) })
+const toHaveSchemas = ( collections, func, ...args ) => ({
+    schemas: collections.map( c => func(c, args) )
+})
 
-const haveSchemaFor = (dbs, allowedSchemaOperations) =>  toHaveSchemas(appendAllowedOperationsToDbs(dbs, allowedSchemaOperations))
+const collectionToHaveReadOnlyCapability = () => expect.objectContaining({ allowedOperations: expect.toIncludeSameMembers(ReadOnlyOperations) })
```

Comment:
`toIncludeSameMembers`  is a matcher from the jest-extended package. 
it checks that the arrays have the same members no matter the order